pimbd/files/pimb.config

```diff
@@ -0,0 +1,23 @@
+# Uncomment the following following section to enable pimb on the interface.
+# option pim     - enable or disable pim
+# option ssbidir - enable or disable ssbidir
+# option mld     - enable or disable mld
+# option igmp    - enable or disable igmp
+# option dev     - set the device name
+
+#config interface 'ETH1'
+#	option pim 'on'
+#	option ssbidir 'on'
+#	option mld 'on'
+#	option igmp 'on'
+#	option dev 'eth1'
+
+
+# Uncomment the following section to configure a rendevous point.
```

Comment:
This may mislead someone completely unaware of openwrt config syntax.

Reply:
I have updated the description of the configuration.